services.go

```
@@ -54,6 +63,47 @@ func applyService(c *cli.Context) error {
 	return nil
 }
 
+func updateServiceYamlContent(content string) string {
+	var serviceType = fetchServiceType(content)
+	switch {
+	case serviceType == GCP:
+		log.Info("Looks like you are creating a service for GCP," +
+			" validating GCP project and bucket now...")
+		if gcpProjectName == "" || gcpProjectName == GCP_PROJECT_NAME_PLACEHOLDER {
+			gcpProjectName = TextInput("Enter a valid GCP project name:")
+		}
+
+		if gcpBucketName == "" || gcpBucketName == GCP_BUCKET_NAME_PLACEHOLDER {
+			gcpBucketName = TextInput("Enter a valid GCP bucket name:")
+		}
+		log.Info("Got your gcp project and bucket info, let's create the service now...")
+		content = replacePlaceholderValues(content, GCP_PROJECT_NAME_PLACEHOLDER, gcpProjectName)
+		content = replacePlaceholderValues(content, GCP_BUCKET_NAME_PLACEHOLDER, gcpBucketName)
+	case serviceType == AWS:
+		log.Info("Looks like you are creating a service for AWS, validating yaml now...")
+	default:
+		fmt.Println("Nothing to update in the yaml.")
```

lets throw an exception/error that the input should be GCP, AWS.